app/controllers/payments_controller.rb

```diff
@@ -0,0 +1,31 @@
+class PaymentsController < ApplicationController
+  def index
```

Comment:
- [ ] A user should not be able to visit the Payment routes without being logged in. Kindly add the `before_action` helper shared earlier

Reply:
Hi @TedLivist 

Thanks a lot for the good review. I would like to disagree. You can't access any categories or payments without being logged in. This is because I have already added the `before_action :authenticate_user!` helper in the `application controller` and it will be available to all other controllers since they all inherit from it. So no need to add that helper in the categories or payments controller(s).

Thanks!